tests/blend.py

```diff
@@ -2,24 +2,34 @@
 
 import pytest
 import numpy as np
-from minerva_lib.blend import to_f32
-from minerva_lib.blend import f32_to_bgr
-from minerva_lib.blend import linear_bgr
+from minerva_lib.blend import make_rgb
+from minerva_lib.blend import clip_image
```

Comment:
Remove tests as function is gone.

Reply:
done `refactor composite_image for single channel`